helm-chart-sources/appscope/templates/webhook-cert-cluster-rbac.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "appscope.fullname" . }}-webhook-cert-cluster-role
+  labels:
+  {{- include "appscope.labels" . | nindent 4 }}
+rules:
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - mutatingwebhookconfigurations
+  verbs:
+  - get
+  - create
+  - patch
+- apiGroups:
+  - certificates.k8s.io
+  resources:
+  - certificatesigningrequests
+  verbs:
+  - create
+  - get
+  - delete
+- apiGroups:
+  - certificates.k8s.io
+  resources:
+  - certificatesigningrequests/approval
+  verbs:
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - create
+  - get
```

Comment:
This is pretty broad. Can we limit this with a `resourceNames` array?

e.g. `resourceNames: ["scope-secret"]`

Reply:
The name here must points to the `signerName` used for https://github.com/criblio/k8s-webhook-cert-manager/ which can be customized in [webhook-cert-manager](https://github.com/criblio/k8s-webhook-cert-manager/releases/tag/v1.0.1) and will be in [Appscope](https://github.com/criblio/appscope/pull/1435/).
@ricksalsa do You have opinion what is the best approach with this limitation?

Reply:
@bdalpe - that makes sense.